projects/plugins/boost/app/modules/cache/Boost_File_Cache.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Page_Cache;
+
+require_once __DIR__ . '/Boost_Cache.php';
```

Comment:
I assume we have this require becase we want this file to load even when the plugin and as a result the composer autoload file isn't loaded. In this case, it might be useful to add a comment so we accidentally do not remove the line.

Reply:
Yeah, that would be an excellent idea. Thanks.